package.json

```diff
@@ -55,6 +55,7 @@
     "client-sessions": "^0.8.0",
     "common-password": "^0.1.2",
     "connect-flash": "^0.1.1",
+    "continuation-local-storage": "^3.2.1",
```

Comment:
Do we need to regenerate and commit `package-lock.json` as well?

Reply:
Whoops @DanailMinchev is right

Reply:
Well spotted. I'll update that now.
Until we use 'npm ci' it's not actually used in the manner we want it to, but we should get into the habit of it none the less.